ros_ws/src/autonomous/wallfollowing2/script/wallfollowing.py

```diff
@@ -207,6 +211,53 @@ def laser_callback(scan_message):
     last_scan = scan_time
 
 
+def dyn_cfg_callback(config, level):
+    rospy.loginfo("""
+        Reconfigure Request:
+          min_throttle: {min_throttle},
+          max_throttle: {max_throttle},
+          radius_lower: {radius_lower},
+          radius_upper: {radius_upper},
+          steering_slow_down: {steering_slow_down},
+          steering_slow_down_dead_zone: {steering_slow_down_dead_zone},
+          max_acceleration: {max_acceleration},
+          corner_cutting: {corner_cutting},
+          straight_smoothing: {straight_smoothing},
+          barrier_size_realtive: {barrier_size_realtive},
+          barrier_size_realtive: {barrier_lower_limit},
+          barrier_upper_limit: {barrier_upper_limit},
+          barrier_exponent: {barrier_exponent},
+          controller_p: {controller_p},
+          controller_i: {controller_i},
+          controller_d: {controller_d}
+    """.format(**config))
+    new_parameters = {
+        'min_throttle': config.min_throttle,
+        'max_throttle': config.max_throttle,
+        'radius_lower': config.radius_lower,
+        'radius_upper': config.radius_upper,
+        'steering_slow_down': config.steering_slow_down,
+        'steering_slow_down_dead_zone': config.steering_slow_down_dead_zone,
+        'high_speed_steering_limit': config.high_speed_steering_limit,
+        'high_speed_steering_limit_dead_zone': config.high_speed_steering_limit_dead_zone,
+        'max_acceleration': config.max_acceleration,
+        'corner_cutting': config.corner_cutting,
+        'straight_smoothing': config.straight_smoothing,
+        'barrier_size_realtive': config.barrier_size_realtive,
+        'barrier_lower_limit': config.barrier_lower_limit,
+        'barrier_upper_limit': config.barrier_upper_limit,
+        'barrier_exponent': config.barrier_exponent,
+        'controller_p': config.controller_p,
+        'controller_i': config.controller_i,
+        'controller_d': config.controller_d}
+    parameters = Parameters(new_parameters)
+    parameters.load()
```

Comment:
This loads the parameters from ROS. It should work as expected if you remove this line.